data/hai/hai missions.txt

```diff
@@ -2099,3 +2099,217 @@ mission "Hai Pilgrimage"
+
+
+
+mission "Acorn Delights: Part 1"
+	minor
+	name "Acorns for Anoo"
+	description "Bring Eeeya to <destination> to deliver <cargo> to her missing granddaughter."
+	source "Allhome"
+	destination "Stonebreak"
+	passengers 1
+	cargo "acorn cookies" 3
+	to offer
+		random < 85
+		has "First Contact: Unfettered: offered"
+	on offer
+		conversation
+			`As you pass by a kitchen supply store in the spaceport, a dishevelled elderly Hai looks towards you with a face of deep concern. She waves you down.`
+			choice
+				`	(See what she wants.)`
+				`	(Ignore her.)`
+					decline
+			`	"My name is Eeeya, and I'm looking for my lost granddaughter, Anoo. She recently graduated from medical school, and I was going to give her three tons of cookies, but she's gone! Please help me find my granddaughter."`
+				goto "actual questions"
+			label questions
+			`	"So please, help me find my granddaughter."`
+			label "actual questions"
+			choice
+				`	"Three tons of cookies!?"`
+					goto tons
+				`	"She's gone?"`
+					goto gone
+				`	"Why don't you ask the authorities to help?"`
+					goto authorities
+				`	"Let's go find her!"`
+					goto accept
+				`	"I have other places to be."`
+					decline
+			label tons
+			`	"Well I... I don't know... I cook when I'm worried, and Anoo has been missing for three months! I've vacuum-sealed all three tons of cookies. They should last a year or two. Maybe she'll share them with her friends? She has many friends! Tons of friends! They all miss her...`
+				goto questions
+			label gone
+			`	"Anoo graduated from a medical academy on Frostmark four months ago, near the top of her class! A prominent hospital on Hai-Home needed a new trauma surgeon and said she would be perfect for the job. She accepted the position and said she would be on vacation for a few weeks.`
+			`	"A month later, she told them she won't be coming, and nobody has heard from her since.`
+				goto questions
+			label authorities
+			`	"I talked to the Central Medical Bureau, and they told me she got clearance for practicing medicine anywhere in Hai space or beyond. They've heard nothing from her since she got clearance. She has to register new prescriptions with the Bureau, so they should have heard from her.`
+			`	"The police told me that several other people have reported Anoo missing, but they refused tell me anything else. They said I had to talk to the military, and the military won't tell me anything.`
+			`	"Maybe she's involved in something secret? Or illegal? No, she'd never do that. Where is she?`
+				goto questions
+			label accept
+			`	"She always talked about doing volunteer work for those who couldn't afford medical treatment. When she was little, her mother would take her to Stonebreak to help at local shelters. Let's start there."`
```

Comment:
Honestly reading this I didnt realise she was coming until the end, mb it could be clearer? Or mb it's just me